pkg/errox/grpc/mapping.go

```diff
@@ -29,6 +29,8 @@ func RoxErrorToGRPCCode(err error) codes.Code {
 		return codes.PermissionDenied
 	case errors.Is(err, errox.NoAuthzConfigured):
 		return codes.Unimplemented
+	case errors.Is(err, errox.ResourceExhausted):
```

Comment:
@jvdm Do you think we should use this code for the enricher stuff instead of Aborted?